gneiss/_model.py

```diff
@@ -80,3 +79,10 @@ def write_pickle(self, filename):
                 pickle.dump(self, fh)
         else:
             pickle.dump(self, filename)
+
+    def percent_explained(self):
+        """ Proportion explained by each principal balance."""
+        # Using sum of squares error calculation (df=1)
+        # instead of population variance (df=0).
+        axis_vars = np.var(self.response_matrix, ddof=1, axis=0)
+        return axis_vars / axis_vars.sum()
```

Comment:
Bumped this up here, because this is relatively agnostic of the underlying model.